modules/AM_ProjectTemplates/language/en_us.lang.php

```diff
@@ -142,5 +142,8 @@
 	'LBL_SCHEDULING_FORM_TITLE' => 'Resource List',
 	'LBL_NONE' => 'None',
 
+	'LBL_AOS_PRODUCTS_AM_PROJECTTEMPLATES_1_FROM_AM_PROJECTTEMPLATES_TITLE' => 'Product: Project Templates from Project Templates Title',
+	'LBL_AM_PROJECTTEMPLATES_PROJECT_1_FROM_PROJECT_TITLE' => 'Project Templates Project from Project Title',
```

Comment:
@horus, fair point, they may be different, i tried to keep everything consistent as possible, however sometimes you can see some differences, both label strings are descriptive enough. Also they totally independent in each module so you don't have to worry about malfunctioning.